### ISSUE TITLE
WP CLI from vendor is not working 

### ISSUE THREAD

**Issue by perruche** [studiometa/create-wordpress-project#17]
WP CLI from composer vendor does not work

```bash
▶ ./vendor/bin/wp plugin list
`autoload.php` not found. Did you run `composer install`?%
```

This is failling at the very first line of `wp-config.php` 
`if ( ! file_exists( dirname( __DIR__ ) . '/vendor/autoload.php' ) )`

This is searching for a file in. :
`./vendor/wp-cli/wp-cli/php/vendor/autoload.php`
and not in `./vendor/autoload.php`

**Comment by titouanmathis** [comment]
See [this comment](https://github.com/studiometa/create-wordpress-project/pull/18#discussion_r464390224) for a detailed explanation of the origin of the bug and how to fix it 😉

**Comment by titouanmathis** [comment]
Fixed in #18.
